nixos/modules/services/web-servers/h2o/default.nix

```diff
@@ -278,6 +284,16 @@ in
           '';
       };
 
+      defaultListenHost = mkOption {
+        type = types.nullOr types.nonEmptyStr;
+        default = null;
+        example = "0.0.0.0";
+        description = ''
+          Adds this to all listen.host blocks by default. Useful for using
```

Comment:
What is "this"? A `host` directive? A listen address? And how does the Dualstack variant of this look like?